### ISSUE TITLE
mountpath

### ISSUE THREAD

**Issue by Simon-CR** [obeone/charts#12]
The configuration data is actually located directly in /etc/dns. The values uses /etc/dns/config which is normally empty.  Note that I'm using pv/pvc and not emptydir.

**Comment by obeone** [comment]
Hi @Simon-CR !

Which chart are you talking about ?

**Comment by Simon-CR** [comment]
sorry about that :) technitium.  

so what I ended up doing to work around that is the following in my values: 

```
persistence:
  config:
    enabled: true
    retain: true
    type: pvc
    existingClaim: pvc-nfs-data-technitium
    mountPath: /etc/dns
```

and my PV is like this: 

```
apiVersion: v1
kind: PersistentVolume
metadata:
  annotations:
    pv.kubernetes.io/provisioned-by: nfs.csi.k8s.io
  name: pv-nfs-data-technitium
spec:
  capacity:
    storage: 10Gi
  accessModes:
    - ReadWriteMany
  persistentVolumeReclaimPolicy: Retain
  storageClassName: nfs-csi
  mountOptions:
    - nfsvers=4.2
  csi:
    driver: nfs.csi.k8s.io
    volumeHandle: <server_ip></export_path>
    volumeAttributes:
      server: <server_ip>
      share: <export_path>
      subDir: technitium
```

thanks

Simon

**Comment by obeone** [comment]
Oh... yes, you're right !
I didn't use (for the moment) this chart... I push a fix

**Comment by obeone** [comment]
fixed in version 1.2.0. 

Sorry about that !
